src/test/java/org/folio/search/controller/SearchInstanceIT.java

```diff
@@ -159,10 +159,16 @@ void streamInstanceIds() throws Exception {
 
       arguments("search by public items.notes.note", "itemPublicNotes all {value}",
         array("bibliographical references"), null),
+      arguments("search by public circulation notes items.notes.note", "itemPublicNotes all {value}",
+        array("first-record"), zeroResultConsumer()),
+      arguments("search by public circulation notes items.notes.note", "itemPublicNotes all {value}",
+        array("SecondRecord"), null),
       arguments("search by private items.notes.note using itemPublicNotes", "itemPublicNotes == {value}",
         array("librarian private note for item"), zeroResultConsumer()),
       arguments("search by private items.notes.note", "items.notes.note == {value}",
         array("Librarian private note for item"), null),
+      arguments("search by circulation notes in items.notes.note", "items.notes.note == {value}",
+        array("testNote"), null),
```

Comment:
can you add a test case that search for `first-record` will return correct result using query
```
items.note.note == "first-record"
```